app/src/main/java/com/bogdan/codeforceswatcher/activity/MainActivity.kt

```diff
@@ -90,8 +117,43 @@ class MainActivity : AppCompatActivity(), SwipeRefreshLayout.OnRefreshListener,
         editor.apply()
     }
 
+    private fun showItemMenu(item: MenuItem, firstTime: Boolean) {
+        val users = CwApp.app.userDao.getAll()
+        if (firstTime) {
```

Comment:
Hm, `firstTime` logic is very strange. Why to not just increment `counter` before calling showItemMenu?